lib/import/mobility.js

```diff
@@ -20,6 +20,24 @@ function yyyymmdd_to_iso(yyyymmdd) {
 }
 
 /**
+ * Return row whether row is valid
+ *
+ * @param{obj} obj - row from mobility file
+ * @return{bool}.
+ */
+function check_row_is_valid(obj) {
+  var missing = Object.keys(obj).find(
+    function(e) {return !obj[e];}
+  );
```

Comment:
Is this just checking that all the values in `obj` are truthy? I think this is a bit nicer using lodash functions: `var missing = !_.every(_.values(row));`. ([docs on _.every](https://lodash.com/docs#every))

However, I think it's better to explicitly check for the empty value. It seems that `fast-csv` sets values to the empty string, so let's check specifically for that instead of relying on Javascript's truthy/falsey behavior. Something along these lines:

```
function check_row_is_valid(row) {
  if (_.every(_.values(row), function(val) { return val !== ''; })) {
    return true;
  } else {
    console.error('row missing some fields:', row);
    return false;
  }
}
```